template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Startup.fs

```diff
@@ -8,33 +8,21 @@ open Microsoft.AspNetCore.Builder
 open Microsoft.AspNetCore.Hosting
 open Microsoft.Extensions.Configuration
 open Microsoft.Extensions.DependencyInjection
-open Microsoft.Extensions.Logging
 
 
 type Startup private () =
 
-    new (env: IHostingEnvironment) as this =
+    new (configuration: IConfiguration) as this =
         Startup() then
-
-        let builder =
-            ConfigurationBuilder()
-                .SetBasePath(env.ContentRootPath)
-                .AddJsonFile("appsettings.json", optional = false, reloadOnChange = true)
-                .AddJsonFile((sprintf "appsettings.%s.json" (env.EnvironmentName)), optional = true)
-                .AddEnvironmentVariables()
-
-        this.Configuration <- builder.Build()
+        this.Configuration <- configuration
```

Comment:
We won't require the Configuration property in the Startup class for authentication in preview2. But it's still kinda nice that it's there to show that it can now come from DI in 2.0.